catalogData/redis32/ha/3x/k8s/sentinel-proxy-service.json

```diff
@@ -0,0 +1,23 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "redis-sentinel-proxy"
+    },
+    "name": "redis-sentinel-proxy"
+  },
+  "spec": {
+    "ports": [
+      {
+        "name": "redis-sentinel-proxy",
+        "port": 6379,
+        "protocol": "TCP"
+      }
+    ],
+    "selector": {
```

Comment:
So probably something more along the lines of `$service_id-proxy` then? while also changing line 27 in the deployment JSON?
https://github.com/18F/kubernetes-broker/pull/55/files#diff-d9cadeb17c3049e9ab78c9687e6ad07bR27

I think I'm going to need to pair on this in the morning for a few minutes to get a better understanding of what I need to do here.